### PR TITLE
Added handling for activities that stretch past a plans duration

### LINKED ISSUE #655: Expansion fails when plan has unfinished simulated activities
When a simulated plan includes unfinished activities (i.e., activities whose effect models don't return before the end time of the plan), command expansion breaks. As far as I can tell, this is because the command expansion server tries to parse the `null` values for end time and duration for the unfinished activity instances and breaks.

Internal ticket: https://jira.jpl.nasa.gov/browse/AERIEQS-439

### PR DESCRIPTION
**Pull request by cohansen**
* **Tickets addressed:** Closes #655
* **Review:** By commit
* **Merge strategy:** Merge (no squash)

## Description
Now if the user is trying to expand and we come across an activity type whose effect model goes past the end of the plan the expansion will have a duration error for that specific activity.

## Verification
I added a test for this scenario alongside our existing tests.

## Documentation
N/A

## Future work
N/A

### CODE REVIEW

**Review by goetzrrGit — APPROVED** [review]
looks great to me.

File: sequencing-server/package-lock.json [hunk 1]
```diff
@@ -1642,6 +1642,17 @@
         "npm": "1.2.8000 || >= 1.4.16"
       }
     },
+    "node_modules/body-parser/node_modules/qs": {
```
Comment (camargo) [inline sequencing-server/package-lock.json]: Do we need this lock file change?

Comment (cohansen) [reply sequencing-server/package-lock.json]: Yeah, it looks like the qs patch version was updated.

Comment (dyst5422) [reply sequencing-server/package-lock.json]: Bummer that body-parser still uses qs despite URLSearchParams for a long while now.

https://dev.to/nerdyman/replacing-query-string-with-native-urlsearchparams-4kdg

**Review by camargo — APPROVED** [review]

**Pull request merged**